static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-always-ask-strategy.js

```diff
@@ -0,0 +1,151 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic-equal-buttons.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features',
+    'common/utils/element-inview'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpicEqualButtons,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             ajax,
+             commercialFeatures,
+             ElementInview) {
+
+    // We want to ensure the test always runs as this enables an easy data lake query to see whether a reader is in the
+    // test segment: check whether the ab_tests field contains a test with name ContributionsEpicAlwaysAskStrategy.
+    // This means having showForSensitive equal to true, and the canRun() function always returning true.
+    // The logic for whether the test-variant is displayed, is handled in the canBeDisplayed() function.
+    return function () {
+        this.id = 'ContributionsEpicAlwaysAskStrategy';
+        this.start = '2016-12-06';
+        this.expiry = '2017-01-06';
+        this.author = 'Guy Dawson';
+        this.description = 'Test to assess the effects of always asking readers to contribute via the Epic over a prolonged period.';
+        this.showForSensitive = true;
+        this.audience = 0.02;
+        this.audienceOffset = 0.85;
```

Comment:
The other test starts at 0.9 and goes through to 1, so if this has an audience of 0.02, it should start at 0.88